spec/services/applicants/remove_from_organisation_spec.rb

```diff
@@ -0,0 +1,72 @@
+describe Applicants::RemoveFromOrganisation, type: :service do
+  subject do
+    described_class.call(
+      organisation: organisation,
+      applicant: applicant,
+      rdv_solidarites_session: rdv_solidarites_session
+    )
+  end
+
+  let!(:organisation) { create(:organisation) }
+  let!(:applicant) { create(:applicant, organisations: [organisation]) }
+  let!(:rdv_solidarites_session) { instance_double(RdvSolidaritesSession) }
+
+  describe "#call" do
+    before do
+      allow(RdvSolidaritesApi::DeleteUserProfile).to receive(:call)
+        .with(
+          user_id: applicant.rdv_solidarites_user_id,
+          organisation_id: organisation.rdv_solidarites_organisation_id,
+          rdv_solidarites_session: rdv_solidarites_session
+        ).and_return(OpenStruct.new(success?: true))
+    end
+
+    it "is a success" do
+      is_a_success
+    end
+
+    it "removes the organisation from the applicant" do
```

Comment:
removes the applicant from the organisation plutôt ? Comme le dit le proverne : "On peut sortir quelqu'un d'une organisation, mais on ne peut jamais sortir l'organisation de quelqu'un"

Reply:
Yes done here : https://github.com/betagouv/rdv-insertion/pull/791/commits/9a7aa76568c984e61d276adffa7d622663f37db5